sample/build.gradle

```diff
@@ -4,7 +4,7 @@ buildscript {
     }
 
     dependencies {
-        classpath "com.android.tools.build:gradle:2.1.2"
+        classpath 'com.android.tools.build:gradle:2.2.2'
```

Comment:
This should go into a separate PR as well (all of them three touching gradle config can however go into the same one).